nhs/data/handling.py

```diff
@@ -29,18 +30,20 @@ def read_csv(file_path: str) -> pl.LazyFrame | None:
 
 @logger.catch()
 @log_entry_exit()
-def read_xlsx(file_path: str) -> pl.LazyFrame | None:
+def read_xlsx(file_path: str, sheet_id: None | int = None) -> dict[str, pl.LazyFrame] | pl.LazyFrame | None:
     """
     Load a .xlsx file into a polars `LazyFrame`, returning None if exception occurs
 
     **NOTE**: Polars use `xlsx2csv` to read .xlsx files, so whole CSV file is read
     """
-    return pl.read_excel(file_path).lazy()
+    frames = pl.read_excel(file_path, sheet_id=sheet_id)
+    if isinstance(frames, dict):
+        return {name: df.lazy() for name, df in frames.items()}
+    return frames.lazy()
 
 
 def __get_spreadsheet_reader(
-    file_extension: str,
-) -> Callable[[str], pl.LazyFrame | None]:
+        file_extension: str) -> Callable[..., pl.LazyFrame | None]:
```

Comment:
Change to `Callable[[str, None | int], dict[str, pl.LazyFrame] | pl.LazyFrame | None]` and add `sheet_id` to the other reader functions as well to make it consistent